package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "parse": "node index.js",
+    "parse": "node App.js",
     "start": "nodemon index.js"
   },
   "devDependencies": {
```

Comment:
Remove all dev. dependencies. Just node-fetch is necessery

Reply:
Done removing the dev dependencies